### ISSUE TITLE
Process finished with exit code -1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by Quebradawill** [milesial/Pytorch-UNet#205]
I have 65 training images, when I train the model, I encountered such errors, what does it mean?

Epoch 1/5: 100%|██████████| 49/49 [21:57<00:00, 26.90s/img, loss (batch)=16.5]
INFO: Checkpoint 1 saved !
Epoch 2/5:   0%|          | 0/49 [00:00<?, ?img/s]['data/imgs\\file70.jpg']
Epoch 2/5:   2%|▏         | 1/49 [00:11<09:19, 11.65s/img, loss (batch)=5.45]['data/imgs\\file64.png']
Epoch 2/5:   2%|▏         | 1/49 [00:24<09:19, 11.65s/img, loss (batch)=36.3]
Process finished with exit code -1073741819 (0xC0000005)

![无标题](https://user-images.githubusercontent.com/3280662/88818654-82ccba00-d1f1-11ea-83a5-df3a34d082cc.png)

**Comment by milesial** [comment]
I don't think this has to do with this repo. Have you reached a memory limit?

**Comment by Quebradawill** [comment]
> I don't think this has to do with this repo. Have you reached a memory limit?

I don't know, how to show memory limit?

**Comment by milesial** [comment]
For RAM, `htop` or `top`.
For VRAM, `nvidia-smi`.

You can look this up in your favorite search engine.

**Comment by wzyjhm** [comment]
INFO: Using device cpu
INFO: Network:
	3 input channels
	1 output channels (classes)
	Bilinear upscaling
INFO: Creating dataset with 40 examples
INFO: Starting training:
        Epochs:          5
        Batch size:      1
        Learning rate:   0.0001
        Training size:   36
        Validation size: 4
        Checkpoints:     True
        Device:          cpu
        Images scaling:  0.5
    
Epoch 1/5:   0%|          | 0/36 [00:00<?, ?img/s]
Process finished with exit code -1073741819 (0xC0000005)

**Comment by milesial** [comment]
Closing, as this issue is now outdated after the recent code refactor.
